.github/container/Dockerfile.triton

```diff
@@ -2,10 +2,21 @@
 ARG BASE_IMAGE=ghcr.io/nvidia/jax-mealkit:jax
 ARG SRC_PATH_TRITON=/opt/openxla-triton
 
+FROM ${BASE_IMAGE} as base
+# Triton setup.py downloads and installs CUDA binaries at specific versions
+# hardcoded in the script itself:
+# https://github.com/openxla/triton/blob/84f9d9de158fb866fac67970f0f5d323999d9db1/python/setup.py#L373-L393
+# Tell Triton to use CUDA binaries from the host container instead. These should be set
+# both during the build stage and in the final container.
+ENV TRITON_PTXAS_PATH=/usr/local/cuda/bin/ptxas
+ENV TRITON_CUOBJDUMP_PATH=/usr/local/cuda/bin/cuobjdump
+ENV TRITON_NVDISASM_PATH=/usr/local/cuda/bin/nvdisasm
+RUN [ -f "${TRITON_PTXAS_PATH}" ] && [ -f "${TRITON_CUOBJDUMP_PATH}" ] && [ -f "${TRITON_NVDISASM_PATH}" ]
```

Comment:
Why use `-f` instead of `-x`?
-f: True if FILE exists and is a regular file.
-x: True if FILE exists and is executable.